extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -311,72 +311,6 @@ public function testRenderInfoboxDataProvider() {
 			[
 				'input' => [
 					[
-						'type' => 'comparison',
```

Comment:
I think it's worth adding a case that handles our main use cases for group with show attribute to keep covering it